eng/pipelines/mgmt.yml

```diff
@@ -0,0 +1,38 @@
+# External variables Settable at queue time:
+# "mavenGoals" : "clean compile".   This is the project object model file for Maven.
+
+trigger:
+  - master
+
+jobs:
+  - job: 'Build'
+
+    strategy:
+      matrix:
+        Java 8:
+          ArtifactName: 'packages'
+          JavaVersion: '1.8'
+        Java 7:
+          ArtifactName: 'packages'
+          JavaVersion: '1.7'
+    
+    pool:
+      vmImage: 'ubuntu-16.04'
+      
+    steps:
+      - task: ShellScript@2
+        displayName: 'call mvn for each mgmt sdk individually'
+        inputs:
+          scriptPath: "$(System.DefaultWorkingDirectory)/eng/pipelines/scripts/mgmt_sdk_compiler.sh"
+          workingDirectory: "$(System.DefaultWorkingDirectory)"
+          failOnStandardError: true
+          args: "$(JavaVersion) ${mavenGoals}"
```

Comment:
If this doesn't need to support anything other then compile I don't think you need to pass the goals all the way through you can directly specify them and eliminate the need to set them in the pipeline web ui.